src/fixup-svg-string.js

```diff
@@ -1,8 +1,11 @@
+const DOMPurify = require('isomorphic-dompurify');
```

Comment:
I don't see this added anywhere in the dependencies.

Also, it looks like `isomorphic-dompurify` is a completely separate project maintained by Some Guy On GitHub(tm), so there's the potential for the DOMPurify version in `isomorphic-dompurify` to become out-of-date.

It looks like all it does is set `window` using JSDOM, which you could just do manually in the tests:
```js
const jsdom = require('jsdom');
const {JSDOM} = jsdom;
const {window} = new JSDOM('<!DOCTYPE html>');
global.window = window;
```